snap-build/snap.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+#
+# Copyright (c) 2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+# Setup the environment and build the snap image.
+# This script runs in the VM.
+
+set -x
+sudo apt-get update -y
+sudo apt-get install -y snapd snapcraft golang-go gcc g++ libcap-ng-dev pkg-config make libz-dev libssl-dev openssl cpio libelf-dev libnewt-dev \
```

Comment:
Could you reformat this list so each package is on a separate line (with a backslash)? Also, you might be able to simplify the list by replacing some of these packages with the `build-essential` package I think.